src/pages/AddScholarship.jsx

```diff
@@ -18,15 +21,11 @@ function AddScholarship() {
           sm={12}
           md={6}
           sx={{ textAlign: { xs: 'center', md: 'left' } }}>
-          <Typography gutterBottom>Submit a Scholarship</Typography>
+          <Typography gutterBottom>{t('addScholarship.sub')}</Typography>
           <Typography variant="h4" gutterBottom>
-            Additional information
-          </Typography>
-          <Typography>
-            Help our team provide the best search results for students and
-            community members looking for scholarships by contributing as much
-            detailed information possible.
+            {t('addScholarship.title')}
           </Typography>
+          <Typography>{t('addScholarship.desc')}</Typography>
```

Comment:
was also thinking of translating the instructions, but I'm not sure if it's necessary since the actual scholarship form is not translated.